src/components/entries/templating/FeelTemplating.js

```diff
@@ -0,0 +1,206 @@
+import { useCallback, useEffect, useState, useMemo, useRef } from 'preact/hooks';
+import { usePrevious, useStaticCallback, useShowEntryEvent } from '../../../hooks';
+import { isFunction } from 'min-dash';
+import { useError } from '../../../hooks';
+import classnames from 'classnames';
+import Description from '../Description';
+import FeelTemplatingEditor from './FeelTemplatingEditor';
+
+const noop = () => { };
+
+/**
+ * @param {Object} props
+ * @param {Object} props.element
+ * @param {String} props.id
+ * @param {String} props.description
+ * @param {Boolean} props.debounce
+ * @param {Boolean} props.disabled
+ * @param {String} props.label
+ * @param {Function} props.getValue
+ * @param {Function} props.setValue
+ * @param {Function} props.validate
+ */
+export default function FeelTemplatingEntry(props) {
```

Comment:
Just to clarify: we do not bring the same optional vs. required behavior as we have in `FEEL` with the new templating, right? Once you use a FEEL templating entry, you can't turn it off.

Reply:
In this first iteration anyways, yeah. But there are plans for this to change relatively soon ([link](https://github.com/camunda/product-hub/issues/163)).